Templates/Databases/MicrosoftSQL/MS SQL SELECT to CSV with SFTP Upload/process.json

```diff
@@ -0,0 +1,83 @@
+﻿{
+	"ProcessTemplates": [
```

Comment:
Just in general - please remove the "Contact support" phrase from error handlers altogether.